Jenkinsfile

```diff
@@ -10,14 +10,14 @@ pipeline {
     parameters {
         booleanParam(name: 'unit_validate', defaultValue: true, description: 'x86 unit tests and vendor check')
         booleanParam(name: 'janky', defaultValue: true, description: 'x86 Build/Test')
-        booleanParam(name: 'experimental', defaultValue: true, description: 'x86 Experimental Build/Test ')
         booleanParam(name: 'z', defaultValue: true, description: 'IBM Z (s390x) Build/Test')
         booleanParam(name: 'powerpc', defaultValue: true, description: 'PowerPC (ppc64le) Build/Test')
         booleanParam(name: 'windowsRS1', defaultValue: false, description: 'Windows 2016 (RS1) Build/Test')
         booleanParam(name: 'windowsRS5', defaultValue: false, description: 'Windows 2019 (RS5) Build/Test')
     }
     environment {
         DOCKER_BUILDKIT     = '1'
+        DOCKER_EXPERIMENTAL = 'y'
```

Comment:
@thaJeztah why `y` ?

Reply:
good one; I think that was what's being used previously 🤔 let me check, otherwise I'll just change to `'1'`

Reply:
@tiborvass fixed